src/main/webapp/script.js

```diff
@@ -98,7 +98,19 @@ function initMap() {
 let selectedElement;
 let selectedIndex;
 
+function shuffleFirstRestaurant(restaurants) {
+  if(restaurants.length === 0) {
+    return restaurants;
```

Comment:
Do we have to return the empty restaurants list?

Reply:
oh thanks for noticing this. i forgot to update this